Cookbook/Technical-Documents/SupportEngineerRole.md

```diff
@@ -9,7 +9,10 @@ When an engineer is on support, these duties have higher priority over Squad dut
 
 * **Update #ios and #ios-usa-support Slack channels topics** The topics should be updated with the iOS Engineers in support for the current day. This update should be done first thing in the morning.
 * **Be available to answer any requests posted in the #ios and #ios-usa-support.** The engineer can either solve the request or delegate to someone who might know how to solve it.
-* **Pick up tickets from the iOS Platform backlog.** These are tickets that don't have any Squad as an owner and still need to be fixed. [Kanban board](https://babylonpartners.atlassian.net/secure/RapidBoard.jspa?rapidView=1100&projectKey=IOSP)
+* **Pick up tickets from the iOS Platform backlog.** These are tickets that don't have any Squad as an owner and still need to be fixed, or ones for which the Platform team might need some help with.
+   * Use this [Kanban board](https://babylonpartners.atlassian.net/secure/RapidBoard.jspa?rapidView=1100&projectKey=IOSP) to see the list of pending BAU tickets
```

Comment:
Is this link working? I can't seem to get to the board. Says "Board not found".

Reply:
It is, yeah. If you can't see it it's probably related to access rights then 🤔  Let's investigate that in Slack

Reply:
+1 used to have access but I get the error too

Reply:
@sdaniel55 @j531 Ok I think I found the culprit, updated the JIRA board's filter yesterday and it seems the filter was private which suddenly made the whole board itself private.

I should have fixed it now, can you retry?

Reply:
@AliSoftware works now 🎉

Reply:
Cool 🎉 
Thanks for testing the link in the first place guys, could easily have stayed unnoticed 😓 (yay JIRA's terrible UX… 😄)